docs-2.0/3.ngql-guide/11.edge-statements/2.update-edge.md

```diff
@@ -0,0 +1,24 @@
+# UPDATE EDGE
+
+```ngql
+UPDATE EDGE <src_vid> -> <dest_vid> [@rank] OF <edge_type> SET <update_columns> [WHEN <condition>] [YIELD <columns>]
+```
+
+Use `UPDATE EDGE` to update properties on an edge. The `UPDATE EDGE` statement only updates one edge type of an edge at a time.
+
+Nebula Graph supports compare-and-set (CAS) operation.
```

Comment:
operations, the plural form
Or just:
Nebula Graph supports compare-and-set (CAS).